config/env.config.js

```diff
@@ -55,5 +55,25 @@ const config = {
     }
 };
 
+const redirect = function(options) { 
```

Comment:
I feel like `redirect` could use being more descriptive (out of context, would you think it's a verb? a noun?..).

Maybe `redirectionAppender` or `redirectionAdder`?  ... OK, neither sounds great either, but I'm sure there's something clever you can come up with :-)